ArticlesDatabase/Sources/ArticlesDatabase/ArticlesTable.swift

```diff
@@ -840,6 +896,34 @@ private extension ArticlesTable {
 		return fetchArticlesWithWhereClause(database, whereClause: whereClause, parameters: parameters)
 	}
 
+	func fetchArticlesBetween(_ webFeedIDs: Set<String>, _ limit: Int?, _ database: FMDatabase, since: Date? = nil, before: Date? = nil) -> Set<Article> {
```

Comment:
As opposed to `fetchUnreadCountBetween`, I kept the positional-style arguments for all but the dates here, so they end up being mixed. Because `since` and `before` are optional, they get moved to the end of the signature.

Could potentially replace uses of `fetchArticlesSince` and `fetchArticles`.